packages/components/inputs/date-range-input/README.md

```diff
@@ -45,24 +45,53 @@ export default Example;
+| `onBlur`               | `Function`<br/>[See signature.](#signature-onBlur)                                           |          |         | Called when the date input loses focus.                                                                                                                                                                       |
+| `id`                   | `string`                                                                                     |          |         | Used as the HTML `id` attribute.                                                                                                                                                                              |
+| `name`                 | `string`                                                                                     |          |         | Used as the HTML `name` attribute.                                                                                                                                                                            |
+| `placeholder`          | `string`                                                                                     |          |         | Placeholder value to show in the input field                                                                                                                                                                  |
+| `isDisabled`           | `boolean`                                                                                    |          |         | Disables the date picker                                                                                                                                                                                      |
+| `isReadOnly`           | `boolean`                                                                                    |          |         | Disables the date picker menu and makes input field read-only                                                                                                                                                 |
+| `hasError`             | `boolean`                                                                                    |          |         | Indicates the input field has an error                                                                                                                                                                        |
+| `hasWarning`           | `boolean`                                                                                    |          |         | Indicates the input field has warning                                                                                                                                                                         |
+
+## Signatures
+
+### Signature `intl`
+
+```ts
+{
+  locale: string;
+  formatMessage: (message: MessageDescriptor) => void;
+}
+```
+
+### Signature `value`
```

Comment:
We have the signature explained in the table above. Maybe remove this sub-heading or moving the explanation over here?

Reply:
This is still related to this [comment](https://github.com/commercetools/ui-kit/pull/2059#discussion_r792494148). 
Btw, the README.md file is automatically generated. 
`yarn generate-readmes`